eclipse-tractusx/bpdm
# feat/doc: Generate Mermaid Persistence Class Diagram from JPA metadata

## Description

Generate Mermaid Persistence Class Diagram from JPA metadata. Add temporary a Rest endpoint to gate and pool to test the generated diagram

Enums are depicted as attributes without reference to the UML Enum Class, Max 15 enum entries are displayed
Not Null Attributes are extended by [1]

@OneToMany elements are depicted as refrence 
@ManyToOne are displayed as attributes without reference. (Mostly they are back references)

## Test instructions

Start gate and pool 
In the Browser (Chrome) call

http://localhost:8080/api/catena/mermaid/
http://localhost:8081/api/catena/mermaid/

Copy result in the mermaid live editor to see result: https://mermaid.live/edit 

Fixes #338 #339
